### DISCUSSION TITLE
Shiny Error - could not find function "tabitem"

### DISCUSSION THREAD

**Discussion by namiraprita** [teamalgoritma/community#254] [Q&A] [answered]
Hi Algoritma Team, 

Saya mengalami error saat ingin membuat feature seperti ini:
![Screen Shot 2022-08-05 at 10 05 17](https://user-images.githubusercontent.com/107253301/182992978-aae15ddc-40c2-4363-8257-e3c1a6979d83.png)

Dengan menjadikan input menjadi RadioButtons menggunakan Tahun dari 2020 - 2022.
Namun, saat mencoba Run App terdapat tulisan :
![Screen Shot 2022-08-05 at 10 07 58](https://user-images.githubusercontent.com/107253301/182993157-9aa296ff-4da4-484a-be13-1fa56ed7eb14.png)

Saat saya check, Tabitem functionnya sudah benar. Apakah ada kesalahan di tempat lainnya?
![Screen Shot 2022-08-05 at 10 08 57](https://user-images.githubusercontent.com/107253301/182993278-a3526465-035b-4a55-a905-ae2737c4cd18.png)

Attached semua file yang dibutuhkan untuk membuka Shiny nya. Terima Kasih!

[ds_dashboard.zip](https://github.com/teamalgoritma/community/files/9264926/ds_dashboard.zip)

**Comment by namiraprita** [comment]
Maaf sebelumnya bisa pakai folder yang ini karena sudah pakai projectnya:
[ds_dashboard.zip](https://github.com/teamalgoritma/community/files/9264928/ds_dashboard.zip)

**Comment by kevbow** [comment, marked as answer]
Halo @namiraprita,

Pada ui.R di line 67 Anda menggunakan fungsi yang salah yaitu `tabitem()` seharusnya `tabItem()` dengan huruf i kapital.

**Reply by namiraprita** [reply]
Thank you! @kevbow
